packages/components/src/Icon.tsx

```diff
@@ -35,3 +35,5 @@ export const Icon = (props: IconProps): ReactNode => {
 export const icon = (icon: IconId, props?: Omit<IconProps, 'icon'>): ReactNode => {
   return <Icon icon={icon} {...props} />;
 };
+
+export const allIconIds = Object.keys(iconLibrary) as IconId[];
```

Comment:
just to avoid duplication if iconLibrary is ever used in additional components, I think it's best to define allIconIds in icon-library.ts and re-export it here.

Reply:
done!